lib/phoenix/endpoint.ex

```diff
@@ -688,12 +688,12 @@ defmodule Phoenix.Endpoint do
 
     * `:websocket` - controls the websocket configuration.
       Defaults to `true`. May be false or a keyword list
-      of options. See "Shared configuration" and
+      of options. See "Common configuration" and
       "WebSocket configuration" for the whole list
 
     * `:longpoll` - controls the longpoll configuration.
       Defaults to `false`. May be true or a keyword list
-      of options. See "Shared configuration" and
+      of options. See "Common configuration" and
```

Comment:
This is the line it is referencing to: https://github.com/phoenixframework/phoenix/pull/3843/files#diff-2e48d48e1c6c3b085dab25613ade742bR720